backend/src/test/java/ch/puzzle/pcts/service/validation/OrganisationUnitValidationServiceTest.java

```diff
@@ -1,245 +1,128 @@
 package ch.puzzle.pcts.service.validation;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.Mockito.*;
 
 import ch.puzzle.pcts.exception.PCTSException;
 import ch.puzzle.pcts.model.error.ErrorKey;
 import ch.puzzle.pcts.model.organisationunit.OrganisationUnit;
 import ch.puzzle.pcts.service.persistence.OrganisationUnitPersistenceService;
 import java.util.Optional;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.provider.Arguments;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
-class OrganisationUnitValidationServiceTest {
-
-    @Mock
-    private OrganisationUnitPersistenceService persistenceService;
+class OrganisationUnitValidationServiceTest
+        extends
+            ValidationBaseServiceTest<OrganisationUnit, OrganisationUnitValidationService> {
 
     @InjectMocks
-    private OrganisationUnitValidationService validationService;
+    private OrganisationUnitValidationService service;
 
-    @DisplayName("Should be successful on validateOnGetById() when id valid")
-    @Test
-    void shouldBeSuccessfulOnValidateOnGetByIdWhenIdIsValid() {
-        Long id = 1L;
+    @Mock
```

Comment:
[MED–CG]: this is against our coding guidelines, move `@Mock` above `@InjectMocks`